### DISCUSSION TITLE
Can't use IS NOT expression in Postgres

### DISCUSSION THREAD

**Discussion by adriangb** [SeaQL/sea-query#792] [General]
Using `Expr::is_not(<expr>, Value::Bool(Some(false)))` generates SQL like `SELECT * FROM table WHERE col IS NOT $1` which is not legal in Postgres (you'd get a `ERROR:  syntax error at or near "$1"` if you do `PREPARE q AS ...`).

I don't see how SeaQuery can easily fix this, but it might be nice to mention in docs and provide alternatives. In particular, hardcoding the RHS as `Expr::expr(SimpleExpr::Constant(Value::Bool(Some(false)))))` works.

**Comment by Huliiiiii** [comment]
You can use `ExprTrait::not_equals`.
